roles/osp_observability/tasks/configure_ceilometer.yml

```diff
@@ -11,11 +11,15 @@
   template:
     src: ceilometer_pipeline.yml.j2
     dest: "{{ ceilometer_pipeline_path }}"
+    owner: root
+    group: root
+    mode: '0644'
 
 - name: Restart the notification agent .
-  shell: 
-    cmd: systemctl restart tripleo_ceilometer_agent_notification.service
+  command: systemctl restart tripleo_ceilometer_agent_notification.service
```

Comment:
Nice! When in doubt, use `command` unless you find you need to use `shell`. I assume the linter suggested not using `shell`.